tests/anndata/test_anndata_ext.py

```diff
@@ -172,6 +172,27 @@ def test_move_to_x(adata_move_obs_mix):
     )
 
 
+def test_move_to_x_copy_uns(adata_move_obs_mix):
+    move_to_obs(adata_move_obs_mix, ["name"], copy_obs=True)
+    adata_move_obs_mix.uns["test"] = "test"
+    new_adata = move_to_x(adata_move_obs_mix, ["name"], copy_uns=True)
+    assert "test" in new_adata.uns
+
+
+def test_move_to_x_copy_obsm(adata_move_obs_mix):
+    move_to_obs(adata_move_obs_mix, ["name"], copy_obs=True)
+    adata_move_obs_mix.obsm["test"] = np.random.rand(adata_move_obs_mix.n_obs, 5)
+    new_adata = move_to_x(adata_move_obs_mix, ["name"], copy_obsm=True)
+    assert "test" in new_adata.obsm
+
+
+def test_move_to_x_copy_varm(adata_move_obs_mix):
+    move_to_obs(adata_move_obs_mix, ["name"], copy_obs=True)
```

Comment:
Doing `copy_obs=True`  here means that "name" is kept as variable; so the `move_to_x` doesn't actually move it back; hence the failing example in my comment was a bit hidden :)